common/lib/capa/capa/templates/choicegroup.html

```diff
@@ -1,53 +1,60 @@
 <form class="choicegroup capa_inputtype" id="inputtype_${id}">
-    <fieldset role="${input_type}group" aria-label="${label}">
-        % for choice_id, choice_description in choices:
-        <label for="input_${id}_${choice_id}"
-            ## If the student has selected this choice...
-            % if input_type == 'radio' and ( (isinstance(value, basestring) and (choice_id == value))  or  (not isinstance(value, basestring) and choice_id in value) ):
+    <fieldset>
+      % if response_data and response_data['label']:
+        <legend id="${id}-legend" class="response-fieldset-legend field-group-hd">${response_data['label']}</legend>
```

Comment:
@ehteshamkafeel unfortunately, we can't have legend have empty text.  That is the accessible name for the radio or checkbox group.    For new problems, We had talked about having warning text show up if a user left the label blank, that said something like "You must specify meaningful question text for this group of radio/checkboxes"  That way the author would be motivated to actually put something meaningful there.  In old problems, there should be a label attribute.  We should use that as the label value. 

Perhaps we can specify some name calculation rules that go something like this:

if label element, legend = label element
else
if label attribute and label attribute == preceding paragraph, legend = label attribute, do not render paragraph //for legacy problems
else
if paragraph exists before group but there is no label element, legend = paragraph text, do not render paragraph //for legacy problems
else
legend = "You must specify meaningful question text for this group of radio/checkboxes"